pyexchange/graph.py

```diff
@@ -0,0 +1,172 @@
+# This file is part of Maker Keeper Framework.
+#
+# Copyright (C) 2020 MikeHathaway
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import time
+import logging
+import requests
+
+from typing import Optional
+from json import JSONDecodeError
+from lib.pymaker.pymaker.util import http_response_summary
+
+get_balances = '''
+  # liquidityPositions(where: {user: {id: "0x332f60EDC783E4Db3E0a18F8dFEB368Ae178CCd9"}}) {
```

Comment:
What is this the address of?  Should it be parameterized?

Reply:
This was for testing balance retrieval. It was parameterized elsewhere, so I removed all of this old testing code.